crawl-ref/source/dat/des/variable/float.des

```diff
@@ -9930,3 +9957,138 @@ x..x..x11xx...........xx11x..x..x
 x1.x.1x11*xxx...{...xxx*11x1.x.1x
 xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
 ENDMAP
+
+NAME:   amcnicky_float_guardroom
```

Comment:
As far as vault authoring goes, the syntax and style is in a good place.

However, the tactics aren't that interesting: the player has plenty of corridors to handle the tougher fight from (and corners!), and while positional magic means some spell users will back off to the open space provided, that's still not great because the monsters will come out of the corridor one at a time.